[BUG] Unable to use with npm

**Describe the bug**
When the dependency is installed with npm, it's not possible to run `npm install -ci` because of the preinstall script.

```
15:37:34.940  [INFO] Running 'npm install -ci' in OMITTED
15:37:37.448  [ERROR] npm ERR! code 1
15:37:37.448  [ERROR] npm ERR! path OMITTED/node_modules/rollup-plugin-sbom
15:37:37.448  [ERROR] npm ERR! command failed
15:37:37.448  [ERROR] npm ERR! command sh -c npx only-allow pnpm
15:37:37.448  [ERROR] npm ERR! npm error code E401
```

**To Reproduce**
Add package as dependency in a npm project.

**Expected behavior**

Package doesn't block dependency installation.

Please see related issues:

https://github.com/JichouP/electron-typescript-ipc/issues/1

https://stackoverflow.com/questions/63165630/trouble-with-npm-preinstall-script

--- a/package.json
+++ b/package.json
@@ -31,7 +31,6 @@
     "build-fixtures": "pnpm --filter \"@fixtures/*\" run build",
     "dev:test": "vitest watch",
     "test": "vitest run --coverage",
-    "preinstall": "npx only-allow pnpm",
     "prepare": "husky",
     "prepublishOnly": "pnpm build",
     "prettier": "prettier -w .",

